README.md

```diff
@@ -20,3 +20,110 @@ with a few commands. Need an ingress to serve traffic? Use Juju to deploy and
+    [`charm.py`](paas_app_charmer/_gunicorn/charm.py) for `gunicorn` based
+    frameworks for integration examples.
+1. Update the [tutorials](docs/tutorials) with the appropriate `fetch-lib`
+  command
+1. Raise a pull request to this repository adding support for the integration.
+1. Add a commented entry for `requires` to all the relevant Charmcraft
+  [templates](https://github.com/canonical/charmcraft/tree/main/charmcraft/templates)
+  for the new integration
+
+### Add a Framework
+
+There are a few recommended steps to add a new framework which we'll go through
+below.
+
+1. Please write a proposal on the
+  [charm topic on discourse](https://discourse.charmhub.io/c/charm/41). This
+  should cover things like:
+  * The programming language and framework you are thinking of
+  * Create an example `rockcraft.yaml` file and build a working OCI image. To
+    see an example for `flask`, install the `latest/edge` version of `rockcraft`
+    and run `rockcraft init --profile flask-framework` and run
+    `rockcraft expand-extensions` and inspect the output.
+  * Create an example `charmcraft.yaml` file and build a working charm. To see
+    an example for `flask`, install the `latest/edge` version of Charmcraft
+    and run `charmcraft init --profile flask-framework` and run
+    `charmcraft expand-extensions` and inspect the output.
+  * How the configuration options of the charm map to environment variables,
+    configurations or another method of passing the information to the app
+  * The requirements and conventions for how users need to configure their app
+    to work with PaaS App Charmer
+  * Which web server to use
+1. Raise a pull request to [rockcraft](https://github.com/canonical/rockcraft)
+  adding a new extension and profile for the framework. This is the flask
+  [profile](https://github.com/canonical/rockcraft/blob/fdd2dee18c81b12f25e6624a5a48f9f1ac9fdb90/rockcraft/commands/init.py#L79)
+  and
+  [extension](https://github.com/canonical/rockcraft/blob/fdd2dee18c81b12f25e6624a5a48f9f1ac9fdb90/rockcraft/extensions/gunicorn.py#L176).
+  The OCI image should work standalone, not just with the charm for the
+  framework.
+1. Raise a pull request to this repository adding a new parent class that can be
+  used by the app charms. The following is the
+  [example for flask](./paas_app_charmer/flask/charm.py).
+1. Raise a pull request to
+  [charmcraft](https://github.com/canonical/charmcraft) adding a new extension
+  and profile for the framework. This is the flask
+  [profile](https://github.com/canonical/charmcraft/tree/main/charmcraft/templates/init-flask-framework)
+  and
+  [extension](https://github.com/canonical/charmcraft/blob/b6baa10566e3f3933cbd42392a0fe62cc79d2b6b/charmcraft/extensions/gunicorn.py#L167).
+1. Write a tutorial and reference documentation for the framework. As an
+  example, this is the flask [tutorial](docs/tutorials/flask.md) and
+  [reference](docs/reference/flask.md) documentation.
```

Comment:
Since the docs were removed from this repo (since they were published elsewhere), the links should be updated for the Flask tutorial and reference pages.

I also think it would be good to have a README section called "Resources" with links to at least some of the docs. The `flask-framework` reference pages would be good to include in that section, and maybe the Rockcraft tutorial.

Reply:
Should be fixed now